types/es-abstract/2015/CreateHTML.d.ts

```diff
@@ -0,0 +1,2 @@
+declare function CreateHTML(string: unknown, tag: string, attribute: string, value?: unknown): string;
```

Comment:
```suggestion
type NonSymbolPrimitive = null | undefined | number | bigint | boolean | string;
declare function CreateHTML(
  string: unknown,
  tag: string,
  attribute: string,
  value?: NonSymbolPrimitive | { toString: (...args: any[]) => string }
): string;
```

Reply:
`unknown` is sufficiently close to that I’d say.

Reply:
right, which is why i think it's important to be explicit. It shouldn't be possible to pass a typecheck when calling this with `value` as a Symbol.

Reply:
(including this in my review; i still think it needs to change - "sufficiently close" is still "not 100% accurate")

Reply:
The biggest issue is that when `bigdecimal` gets added, the `NonSymbolPrimitive` type definition will need to be updated, whereas `unknown` will automatically include `bigdecimal`.

Reply:
I think that's a reasonable compromise, especially for this library, which won't need to support bigdecimals until the ES edition it lands in (if it does).